packages/graphiql-toolkit/src/storage/custom.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * This function enables a custom namespace for localStorage
+ */
+
+import { Storage } from './base';
+
+export function createLocalStorage(namespace: string): Storage {
```

Comment:
I still would prefer this to be an object, so it's easier to expand on, and we don't have to worry about ordered arguments as further options are added.
```suggestion
export CreateLocalStorageOptions = {
  /**
   * specify a different storage namespace prefix from the default of 'graphiql' 
   */
  namespace?: string
}
/**
 * generate a custom local storage adapter for GraphiQL `storage` prop.
 */
export function createLocalStorage({ namespace }: ): Storage {
```

either way, this library should have some API docs added like this!